### ISSUE TITLE
Accessibility: vertical table header - row scope

### ISSUE THREAD

**Issue by Ferrisx4** [tripal/tripal#1208]
## Feature Request/Discussion

* This feature does attempt to solve an existing problem with Tripal
* I am open to developing or collaborating on an extension module if this is not a good fit for Tripal Core

### Description

Various tables generated by Tripal, for instance the "Summary" section on an Organism page, utilize a vertical header. For the sake of accessibility, (screen readers, etc.), defining the scope of a row is required ([see the w3 entry here](https://www.w3.org/WAI/WCAG21/Techniques/failures/F91.html)).

For example: 
```html
<tr>
  <th scope="row">Resource Type</th>
  <td>Organism</td>
</tr>
```
![Screen Shot 2021-05-28 at 3 09 17 PM](https://user-images.githubusercontent.com/9593623/120031362-b53c7480-bfc6-11eb-8dc5-e36b011462b4.png)

### Your Specific Use Case

It is required as per accessibility standards.

### Generally Applicable

All Tripal sites should take accessibility standards into account (see below). 

### Additional information

Accessibility standards are important to adhere to and make use of, and not just for passing tests.  Through recent scans of a Tripal site that is currently in development, there are a number of issues that may fall to Tripal core to implement. Therefore, I also suggest a new label for Tripal issues be created - "accessibility". Most of these will also be "enhancements", an existing label, but it will be useful to highlight these.

**Comment by guignonv** [comment]
I agree with you: accessibility, even if it requires more work, should be taken into account.

**Comment by spficklin** [comment]
I labeled this a release blocker as it's a requirement for i5k before then can upgrade.

**Comment by laceysanderson** [comment]
Moving this discussion over to Tripal 4 so it's addressed when we upgrade the page display.

**Comment by Ferrisx4** [comment]
> Moving this discussion over to Tripal 4 so it's addressed when we upgrade the page display.

Hi @laceysanderson, I'm having trouble finding where this ended up, can you lend a hand?
